Sources/FileSystem/Model/BetaGroup.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+
+public struct BetaGroup: Codable, Equatable {
+
+    public typealias EmailAddress = String
+
+    public var id: String?
```

Comment:
Is this nil because we won't have an id until we `push` it?

Reply:
yes, when creating from dish the id will be nil